odoo/addons/base/models/res_config.py

```diff
@@ -536,6 +579,25 @@ def execute(self):
                 groups.write({'implied_ids': [(3, implied_group.id)]})
                 implied_group.write({'users': [(3, user.id) for user in groups.mapped('users')]})
 
+        # config fields: store ir.config_parameters
+        SuIrConfigParameter = self.env['ir.config_parameter'].sudo()
+        for name, icp in classified['config']:
+            value = self[name]
+            if value and self._fields[name].type == 'char':
+                # storing developer keys as ir.config_parameter may lead to nasty
+                # bugs when users leave spaces around them
+                value = value.strip()
```

Comment:
What if `value == " "`, do you want to store `""` or `False`?
I suggest to avoid the `if value` in the condition, and use an expression that works in all cases, like
```python
value = (value or "").strip() or False
```